main.go

```diff
@@ -155,19 +174,30 @@ func main() {
 		log.Fatalf("%+v", errors.WithStack(err))
 	}
 
-	if output == "stdout" {
+	if !hasChange && *listFileName {
+		return
+	}
+	if hasChange && *listFileName && output != "write" {
+		fmt.Println(filePath)
+	} else if output == "stdout" || filePath == reviser.StandardInput {
```

Comment:
`else if output == "stdout" || filePath == reviser.StandardInput` -> `else if output == "stdout" && filePath == reviser.StandardInput`

Reply:
Line 182 uses an "or", for these 2 cases:
- we want to print the formatted output to stdout when the option says so (-output stdout)
- we must print it to the stdout when the filePath is stdin, regardless of the "-output stdout" option (otherwise we don't know where to save it)